fdbrpc/Stats.h

```diff
@@ -57,11 +57,13 @@ struct ICounter {
 
 template <>
 struct Traceable<ICounter*> : std::true_type {
-	static std::string toString(ICounter const* counter) {
+	static TraceValue toTraceValue(ICounter const* counter) {
 		if (counter->hasRate() && counter->hasRoughness()) {
-			return format("%g %g %lld", counter->getRate(), counter->getRoughness(), (long long)counter->getValue());
+			return TraceValue(
+			    format("[%g,%g,%lld]", counter->getRate(), counter->getRoughness(), (long long)counter->getValue()),
```

Comment:
Does this change the format in XML? That might be a little undesirable.

Reply:
Yes, this changes the format in XML. It still produces valid XML, but I can push this logic down to the `ITraceLogFormatter` if the current change breaks existing tooling.